homeassistant/components/light/yeelight.py

```diff
@@ -604,13 +611,14 @@ def transitions_config_parser(transitions):
 
         return transition_objects
 
-    def start_flow(self, transitions, count=0):
+    def start_flow(self, transitions, action=ACTION_RECOVER, count=0):
```

Comment:
Should action follow the count instead of this way? The schema above has count as the first parameter.

Reply:
Changed order.